Experiments/Experiments/DefaultFeatureFlagService.swift

```diff
@@ -42,7 +42,7 @@ public struct DefaultFeatureFlagService: FeatureFlagService {
         case .backgroundProductImageUpload:
             return true
         case .consolidatedCardReaderManuals:
-            return buildConfig == .localDeveloper || buildConfig == .alpha
+            return true
```

Comment:
I just realized I can remove this entirely, right? After this, there's nothing behind the feature flag.

Reply:
Gotcha! Removed on https://github.com/woocommerce/woocommerce-ios/pull/7178/commits/71e09314be1d120bf6e7274de0156adf050c22e7